src/player.ts

```diff
@@ -385,6 +397,7 @@ export default class Player extends EventEmitter {
   gaiaPhase() {
     /* Move gaia power tokens to regular power areas */
     // Terrans move directly to power area 2
+    // for Terrans is done  after gaia Income selection
     if (this.faction === Faction.Terrans) {
       this.data.power.area2 += this.data.power.gaia;
       if (this.data.brainstone === BrainstoneArea.Gaia ) {
```

Comment:
This test is always false, terrans have no brain stone.

Reply:
yes, I thought I already corrected this ...